### ISSUE TITLE
请问为什么model里面不含激活函数？

### ISSUE THREAD

**Issue by YiminXu1996** [eecn/Hyperspectral-Classification#9]
您好，

我看了您的model里面的结构发现最后一层全连接层不含激活函数，请问是故意这么设计的还是有别的含义？

谢谢

**Comment by eecn** [comment]
CrossEntropyLoss combines nn.LogSoftmax() and nn.NLLLoss() in one single class.For a classification problem , if you want specify activate function,use func:`nn.NLLLoss` repalce class:'CrossEntropyLoss'.ref to [CrossEntropyLoss](https://pytorch.org/docs/stable/_modules/torch/nn/modules/loss.html#CrossEntropyLoss).
